_posts/2024-09-15-labels-talk/labels-talk.Rmd

```diff
@@ -0,0 +1,158 @@
+| 2022 | ^[I was elated to be invited to co-instruct the WTF workshop for the first time;  due to the amount of preparation required I decided not to submit an abstract.]Washington, D.C. | ✔️        | ✔️                   |                |               |
+| 2023 | ^[I was bummed to not being giving a talk, but also still really enjoyed co-instructing the WTF workshop.]Chicago          | ✔️        | ✔️                   | ✔️             |               |
+| 2024 | ^[First time giving a talk! I actually ended up giving two talks this week because a colleague on the R-Ladies leadership team was unable to give her talk in person last minute.]Seattle          | ✔️        | ✔️                   | ✔️       |   ️✔️   |
+
+
+
+
+## Posit submission and review process
+
+Submitting a talk to posit::conf requires both an abstract and a one minute
+video recording^[I do not have my older video recordings, so I did not include
+the submitted video recordings in this post.]. I am grateful that the posit::conf(2023) program committee wrote the blog post [How we build the posit::conf() program](https://posit.co/blog/how-we-build-the-positconf-program/). To paraphrase,
+the committee is looking for talks that
+
+1. contain actionable and impactful insights, and
+
+2. are of broad interest.
+
+In addition, the post provides examples of strong video submissions, which
+are filmed in interesting locations, with props or engaging media, and 
+provide compelling stories.
+
+This program committee's post was published after my 2021 and 2023 rejections. Understanding
+how the process works made it easier to not take the rejections personally, and to
+try to figure out how to frame a new submission. 
+
+## 2021 -- rejected
+
+**In R:** At this point I was working in market research, where my daily workflow
+included importing SPSS labelled data sets into R and doing subsequent wrangling 
+and analysis. I had also recently started blogging, but I had not yet 
+blogged about labelled data at the time of the submission.
+
+**Title:** _Leveraging labelled data in R_
+
+**Abstract:** If you are at rstudio::global(2021), the thought of working with SPSS or SAS data sets may make you cringe. However, with R’s haven, labelled, and sjlabelled packages, you can leverage the inherent data labelling structure in these data sets to put metadata about your data (variable labels, value labels) right at your fingertips, making it easier to navigate data while also allowing the user to convert metadata to data. In this presentation, I will discuss general characteristics of labelled data, importing labelled data in R, storage of metadata as attributes, and practical tips for data analysis with labelled data. I will also address some potentially unexpected consequences when working with labelled data. By the end of the presentation, you may be wishing that your R workflow started with labelled data!
+
+**Email response:**
```

Comment:
because the response is so generic I wonder if it's worth including it? (I have that exact same email in my inbox history :joy_cat: )

Reply:
you are right, probably not worth including. i have removed both email responses, and added a in the review process

The email responses to submissions from Posit are
kind but generic (so there is no personalized feedback), and I am grateful that the posit::conf(2023)...